examples/ignition/coreos-install.yaml

```diff
@@ -0,0 +1,19 @@
+---
+ignition_version: 1
+systemd:
+  units:
+    - name: install.service
+      enable: true
+      contents: |
+        [Unit]
+        Requires=network-online.target
+        After=network-online.target
+        [Service]
+        Type=oneshot
+        ExecStart=/usr/bin/coreos-install -d /dev/sda -C {{.coreos_channel}} -V {{.coreos_version}}
+        ExecStart=/usr/bin/udevadm settle
```

Comment:
Ok, you can leave all of this off. `coreos-install` will support Ignition in the next OS release.